lib/protoparser/opentelemetry/stream/streamparser.go

```diff
@@ -267,7 +267,7 @@ func appendAttributesToPromLabels(dst []prompbmarshal.Label, attributes []*pb.Ke
 	for _, at := range attributes {
 		dst = append(dst, prompbmarshal.Label{
 			Name:  sanitizeLabelName(at.Key),
-			Value: at.Value.FormatString(),
+			Value: at.Value.FormatString(true),
```

Comment:
This change shouldn't affect VictoriaMetrics.

Reply:
Hey @AndrewChubatiuk @f41gh7: shall we discuss this or is it clear how to proceed?

Reply:
VictoriaMetrics should adopt this change as well. It'll pretty metrics format.

I think we should mention it at changelog as well.

Reply:
updated changelog

Reply:
Thanks, I discovered panic while was adding unit tests.

Reply:
See this commit https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8387/commits/3cfcec0a1c2c19f0c6ea09b962d2441289978278